### PR TITLE
Describe support for DID metadata properties

### PR DESCRIPTION
**Pull request by peacekeeper**
Partially addresses https://github.com/trustoverip/tswg-did-method-webs-specification/issues/11.

### CODE REVIEW

File: spec/did_metadata.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+  "didDocument": {
+    "id": "did:webs:example.com:Ew-o5dU5WjDrxDBK4b4HrF82_rYb6MX6xsegjq4n0Y7M"
+    // ... other properties
+  },
+  "didResolutionMetadata": {
+  },
+  "didDocumentMetadata": {
+    "equivalentId": [
+      "did:webs:foo.com:Ew-o5dU5WjDrxDBK4b4HrF82_rYb6MX6xsegjq4n0Y7M",
+      "did:webs:bar.com:Ew-o5dU5WjDrxDBK4b4HrF82_rYb6MX6xsegjq4n0Y7M"
```
Comment (2byrds) [inline spec/did_metadata.md:108]: Perhaps include a port difference example as well

**Review by 2byrds — APPROVED** [review]
one optional suggested addition

File: spec/did_metadata.md (hunk 1 above)
Comment (peacekeeper) [reply spec/did_metadata.md:109]: Good idea, addressed in https://github.com/trustoverip/tswg-did-method-webs-specification/pull/49/commits/5ed733235ffc2854f546fcab77375b1aef70b562

**Review by swcurran — APPROVED** [review]
LGTM

**Pull request merged**